### ISSUE TITLE
forced inclusion of cascading-core and hadoop-core

### ISSUE THREAD

**Issue by sorenmacbeth** [nathanmarz/dfs-datastores#23]
When I use dfs-datastores in a leiningen project and build an uberjar, dfs-datastores causes cascading-core 2.0.7 and hadoop-core 1.0.3 to be included in the jar.

I have to declare the deps like so to exclude them.

``` clojure
[com.backtype/dfs-datastores-cascading "1.3.0"
                  :exclusions [cascading/cascading-core
                               org.apache.hadoop/hadoop-core]]
```

**Comment by sritchie** [comment]
Hey, I bet we can fix this with a "provided" dependency. What do you think?

**Comment by sorenmacbeth** [comment]
Yeah, that's what I was thinking. I've no idea how to do that with sbt, but
I think that's probably the way to go.

On Mon, Mar 25, 2013 at 11:40 AM, Sam Ritchie notifications@github.comwrote:

> Hey, I bet we can fix this with a "provided" dependency. What do you think?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/nathanmarz/dfs-datastores/issues/23#issuecomment-15412979
> .

## 

http://about.me/soren
